packages/postgres-database/src/simcore_postgres_database/models/folders_v2.py

```diff
@@ -0,0 +1,85 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .workspaces import workspaces
+
+folders_v2 = sa.Table(
+    "folders_v2",
```

Comment:
In the future, after the migration is complete, I would drop the v2

Reply:
yes I will drop it, once everything is working and everybody is satisfied.